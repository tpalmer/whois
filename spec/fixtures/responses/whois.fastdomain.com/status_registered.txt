The data in the FastDomain Inc. WHOIS database is provided
to you by FastDomain Inc. for information purposes only,
that is, to assist you in obtaining information about or related to
a domain name registration record. FastDomain Inc. makes
this information available "as is," and does not guarantee its
accuracy.  By submitting a WHOIS query, you  agree that you will use
this data only for lawful purposes and that, under no circumstances
will you use this data to: (1) allow, enable, or otherwise support
the transmission of mass unsolicited, commercial advertising or
solicitations via direct mail, electronic mail, or by telephone; or
(2) enable high volume, automated, electronic processes that apply
to FastDomain Inc. (or its systems). The compilation,
repackaging, dissemination or other use of this data is expressly
prohibited without the prior written consent of
FastDomain Inc.. FastDomain Inc. reserves the
right to modify these terms at any time.  By submitting this query,
you agree to abide by these terms.

=-=-=-=

Registrar: FastDomain Inc.
Provider Name....: FastDomain Inc.
Provider Whois...: whois.fastdomain.com
Provider Homepage: http://www.fastdomain.com/

Domain Name: FASTDOMAIN.COM

   Created on..............: 2002-11-07 05:49:33 GMT
   Expires on..............: 2017-11-07 05:49:33 GMT
   Last modified on........: 2012-04-19 17:40:46 GMT

Registrant Info: (FAST-18447012)

   Domain Privacy Service
   1958 South 950 East
   Provo, Utah -
   United States
   Phone: +1.8014948462
   Fax..:
   Email: whois@fastdomain.com
   Last modified: 2012-06-11 20:32:05 GMT

Administrative Info: (FAST-18447012)

   Domain Privacy Service
   1958 South 950 East
   Provo, Utah -
   United States
   Phone: +1.8014948462
   Fax..:
   Email: whois@fastdomain.com
   Last modified: 2012-06-11 20:32:05 GMT

Technical Info: (FAST-18447012)

   Domain Privacy Service
   1958 South 950 East
   Provo, Utah -
   United States
   Phone: +1.8014948462
   Fax..:
   Email: whois@fastdomain.com
   Last modified: 2012-06-11 20:32:05 GMT

Status: Active

Domain servers in listed order:

   NS1.FASTDOMAIN.COM
   NS2.FASTDOMAIN.COM
   =-=-=-=

Now UNLIMITED Storage and UNLIMITED Bandwidth and Host UNLIMITED Domains on one account for ONLY $6.95 per month.
You also get a *FREE* DOMAIN REGISTRATION for one year when you host with http://www.fastdomain.com/
