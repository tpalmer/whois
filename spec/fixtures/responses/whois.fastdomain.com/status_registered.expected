#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2002-11-07 05:49:33 GMT")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-04-19 17:40:46 GMT")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2017-11-07 05:49:33 GMT")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "FastDomain"
  should: %s.organization == "FastDomain Inc."
  should: %s.url          == "http://www.fastdomain.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id           == "FAST-18447012"
  should: %s[0].name         == "Domain Privacy Service"
  should: %s[0].organization == nil
  should: %s[0].address      == "1958 South 950 East"
  should: %s[0].city         == "Provo"
  should: %s[0].state        == "Utah"
  should: %s[0].zip          == "-"
  should: %s[0].country      == "United States"
  should: %s[0].phone        == "+1.8014948462"
  should: %s[0].fax          == nil
  should: %s[0].email        == "whois@fastdomain.com"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id           == "FAST-18447012"
  should: %s[0].name         == "Domain Privacy Service"
  should: %s[0].organization == nil
  should: %s[0].address      == "1958 South 950 East"
  should: %s[0].city         == "Provo"
  should: %s[0].state        == "Utah"
  should: %s[0].zip          == "-"
  should: %s[0].country      == "United States"
  should: %s[0].phone        == "+1.8014948462"
  should: %s[0].fax          == nil
  should: %s[0].email        == "whois@fastdomain.com"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id           == "FAST-18447012"
  should: %s[0].name         == "Domain Privacy Service"
  should: %s[0].organization == nil
  should: %s[0].address      == "1958 South 950 East"
  should: %s[0].city         == "Provo"
  should: %s[0].state        == "Utah"
  should: %s[0].zip          == "-"
  should: %s[0].country      == "United States"
  should: %s[0].phone        == "+1.8014948462"
  should: %s[0].fax          == nil
  should: %s[0].email        == "whois@fastdomain.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(2)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.fastdomain.com"
  should: %s[0].ipv4 == nil
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.fastdomain.com"
  should: %s[1].ipv4 == nil
